Student_t.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
```

Comment:
<p>I guess changing<code>range</code> and <code>number</code> in interactive demo does no  make sense. Just think about it, if you want to understand the property of laplace, Gaussian, Student-t distributions and comparison between them, then what you will change? Maybe you can go through the text of this figure, try to understand the concept behind this figure, and then start working on this. This will give you sense, what should be variable in demo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>